com.unity.ml-agents/CHANGELOG.md

```diff
@@ -46,26 +46,26 @@ and this project adheres to
 #### ml-agents / ml-agents-envs / gym-unity (Python)
 - The `--load` and `--train` command-line flags have been deprecated. Training
   now happens by default, and use `--resume` to resume training instead. (#3705)
-- The Jupyter notebooks have been removed from the repository.
+- The Jupyter notebooks have been removed from the repository. (#3704)
 - Removed the multi-agent gym option from the gym wrapper. For multi-agent
-  scenarios, use the [Low Level Python API](../docs/Python-API.md).
+  scenarios, use the [Low Level Python API](../docs/Python-API.md). (#3681)
 - The low level Python API has changed. You can look at the document
   [Low Level Python API documentation](../docs/Python-API.md) for more
   information. If you use `mlagents-learn` for training, this should be a
-  transparent change.
+  transparent change. (#3681)
 - Added ability to start training (initialize model weights) from a previous run
   ID. (#3710)
 - The GhostTrainer has been extended to support asymmetric games and the
-  asymmetric example environment Strikers Vs. Goalie has been added.
+  asymmetric example environment Strikers Vs. Goalie has been added. (#3653)
 - The `UnityEnv` class from the `gym-unity` package was renamed
   `UnityToGymWrapper` and no longer creates the `UnityEnvironment`.
   Instead, the `UnityEnvironment` must be passed as input to the
-  constructor of `UnityToGymWrapper`
+  constructor of `UnityToGymWrapper` (#3812)
 
 ### Minor Changes
 
 #### com.unity.ml-agents (C#)
-- `StackingSensor` was changed from `internal` visibility to `public`
+- `StackingSensor` was changed from `internal` visibility to `public`. (#3701)
 - Academy.InferenceSeed property was added. This is used to initialize the
   random number generator in ModelRunner, and is incremented for each ModelRunner. (#3823)
 - Updated Barracuda to 0.6.3-preview.
```

Comment:
Didn't update this line, because it's going away in https://github.com/Unity-Technologies/ml-agents/pull/3875/files